perfkitbenchmarker/vm_util.py

```diff
@@ -661,14 +661,14 @@ def NamedTemporaryFile(prefix='tmp', suffix='', dir=None, delete=True):
       os.unlink(f.name)
 
 
-def GenerateSSHConfig(vms):
+def GenerateSSHConfig(benchmark_spec):
   """Generates an SSH config file to simplify connecting to "vms".
 
   Writes a file to GetTempDir()/ssh_config with SSH configuration for each VM in
   'vms'.  Users can then SSH with 'ssh -F <ssh_config_path> <vm_name>'.
```

Comment:
nit: update documentation to mention group based access - if this is documented anywhere else (like the readme), please update that as well